ios/Classes/Models/SchibstedAccountEvent.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+struct SchibstedAccountEvent: Codable {
+    let schibstedAccountLoginState: String
```

Comment:
Would be great to not have these prefixes (`schibstedAccount`) for each variable. However I don't have knowledge if these are required when sending json, if yes then please discard this comment.

Reply:
the prefix added for sake of consistency with android version